code/game/turfs/simulated/floor/plasteel_floor.dm

```diff
@@ -438,6 +438,8 @@
 /turf/simulated/floor/plasteel/sepia
 	icon_state = "sepia"
 
+/turf/simulated/floor/plasteel/sandy
+	icon_state = "sandy"
 

```

Comment:
Actually, this would baseturf to /beach/sand
beach/sand would baseturf to beach/sand.